dali/test/python/test_operator_readers_numpy.py

```diff
@@ -314,7 +314,7 @@ def _testimpl_numpy_reader_roi(file_root, batch_size, ndim, dtype, device, fortr
         assert_array_equal(roi_arr, sliced_arr)
 
 # testcase name used for visibility in the output logs
-def _testimpl_numpy_reader_roi_empty_axes(testcase_name, file_root, batch_size, ndim, dtype, device, file_filter="*.npy"):
+def _testimpl_numpy_reader_roi_empty_axes(testcase_name, file_root, batch_size, ndim, dtype, device, fortran_order, file_filter="*.npy"):
```

Comment:
fortran_order is not used in the test code (only in the files generated before), but it's kept in the arguments so that nosetests displays that its running this testcase.